### DISCUSSION TITLE
IBM Datastage XML edit assembly doesn't work with Harmann flash replacement plugin

### DISCUSSION THREAD

**Discussion by AbeChamy** [airsdk/Adobe-Runtime-Support#902] [General]
Hi, 

IBM DataStage is a client that is installed on Microsoft windows and it uses Adobe Flash.
Since Adobe flash is out of support , I contacted IBM support and they recommended to install a plug-in supplied by harman .
This is Harman reply to us :
IBM certified the hierarchical datastage on  the Harman supplied plug-in , which is a replacement and supported version of the Flash Player past 31 december 2020. For more information see https://services.harman.com/partners/adobe.
The plug-in did not work, and after consulting Microsoft and IBM on that matter , they did not help.
Please help us install the Harman plugin for flash replacement on our WIN 10 pc's

**Comment by ajwfrost** [comment]
> The plug-in did not work
Can you email me at andrew.frost at harman.com? This works for a number of customers, so there may be some other factor that's preventing it. I would also ensure you've checked with your IT departments as we've found a number of companies who have just blocked all Flash Player capabilities in Windows via some group policies..

thanks
